test/s1/appframework/appframework_test.go

```diff
@@ -143,6 +143,38 @@ var _ = Describe("s1appfw test", func() {
 			//Verify Apps are installed
 			testenv.VerifyAppInstalled(deployment, testenvInstance, testenvInstance.GetName(), []string{podName}, appListV2, true, "enabled", true, false)
 
+			// Scale Standalone instance
+			testenvInstance.Log.Info("Scaling Up Standalone CR")
+			scaledReplicaCount := 2
+			standalone = &enterprisev1.Standalone{}
+			err = deployment.GetInstance(deployment.GetName(), standalone)
+			Expect(err).To(Succeed(), "Failed to get instance of Standalone")
+
+			standalone.Spec.Replicas = int32(scaledReplicaCount)
+
+			err = deployment.UpdateCR(standalone)
+			Expect(err).To(Succeed(), "Failed to scale up Standalone")
+
+			// Ensure standalone is scaling up
+			testenv.VerifyStandalonePhase(deployment, testenvInstance, deployment.GetName(), splcommon.PhaseScalingUp)
+
+			// Wait for Standalone to be in READY status
+			testenv.VerifyStandalonePhase(deployment, testenvInstance, deployment.GetName(), splcommon.PhaseReady)
+
+			// Wait for Monitoring Console Pod to be in READY status
+			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+
+			podNames := []string{fmt.Sprintf(testenv.StandalonePod, deployment.GetName(), 0), fmt.Sprintf(testenv.StandalonePod, deployment.GetName(), 1)}
```

Comment:
nit: probably create a function to get the podNames, something like `getPodNames`, pass the number of replicas as argument to it and it will run a loop over the nunber of replicas and will eventually return a slice of string with podnames.

Reply:
Yeap makes sense however there is already a method to getpodnames which return all pods in the namespace and used by multiple test cases. Maybe that can be enhanced to filter based on replicas or KIND. Will take this as part of test framework enhancement effort in future.